mongo/tests/test_config.py

```diff
@@ -18,3 +20,13 @@ def test_deprecated_schema(instance):
     instance['hosts'] = ['mongodb+srv://sandbox.foo.bar.mongodb.com:27017']
     with pytest.raises(ConfigurationError, match='Could not build a mongo uri with the given hosts'):
         MongoConfig(instance, mock.Mock())
+
+
+def test_hosts_can_be_singular(instance):
+    instance['hosts'] = 'localfoost'
+    check = MongoDb('mongo_check', {}, instances=[instance])
```

Comment:
Is there any reason to not test this directly on `MongoConfig`?

Reply:
That's a good question, the reason is I need to instantiate a MongoDb anyway to test the `load_configuration_models` and check that the generated model (`_config_model_instance`) also applies the same logic